index.ts

```diff
@@ -1,3 +1,3 @@
 export * from "./typechain";
 export * from "./src/DeploymentUtils";
-export * from "./utils/MerkleTree";
+export * from "./utils";
```

Comment:
This will allow `constants` to be exported so we can import from `sdk-v2` and re-export